field/TinyLang/Field/Existential.hs

```diff
@@ -0,0 +1,23 @@
+module TinyLang.Field.Existential
+    ( Some (..)
```

Comment:
Great that you've extracted this to a separate module. We'll eventually need to make a library out of the [much nicer stuff](https://github.com/input-output-hk/plutus/blob/master/language-plutus-core/src/Language/PlutusCore/Universe/Core.hs) that I've implemented in Plutus recently and you it here as well.